core_backend/app/contents/models.py

```diff
@@ -87,6 +96,25 @@ async def save_content_to_db(
         "generation_name": "save_content_to_db",
     }
 
+    if CHECK_CONTENT_LIMIT:
+        # get content_quota value for this user from UserDB
+        user_db = await get_user_by_userid(user_id=user_id, asession=asession)
```

Comment:
Why get the whole user and not just the `content_quota` for the user? Less data to move and SQL ops are pretty quick.

Reply:
Fair point, updating